modules/ROOT/pages/sysconfig-enabling-wifi.adoc

```diff
@@ -58,6 +58,7 @@ systemd:
         RemainAfterExit=yes
         ExecStart=/usr/bin/rpm-ostree install --apply-live --allow-inactive NetworkManager-wifi iwlwifi-dvm-firmware
         ExecStart=/bin/touch /var/lib/%N.stamp
+        ExecStartPost=/usr/bin/systemctl restart NetworkManager.service
```

Comment:
Hmm, feels like for a oneshot service, there isn't much difference between `ExecStart=` and `ExecStartPost=`? Fine as is too though!

Reply:
That said, did you try `systemctl reload NetworkManager.service` instead? If that works, that should be preferred I think.

Reply:
I tried `systemctl reload NetworkManager.service` but it didn't work.